setup.py

```diff
@@ -6,4 +6,7 @@
     description="Miscellaneous utilities useful for interaction with CDIS systems.",
     license="Apache",
     packages=["cdisutils"],
+    install_requires=[
```

Comment:
yeah it was not working when provisioning a new VM with tungsten, that's why I added it here